operator/controllers/seldondeployment_prepackaged_servers.go

```diff
@@ -107,7 +111,12 @@ func addTFServerContainer(r *SeldonDeploymentReconciler, pu *machinelearningv1.P
 
 		}
 
-		_, err := InjectModelInitializer(deploy, tfServingContainer.Name, pu.ModelURI, pu.ServiceAccountName, pu.EnvSecretRefName, r)
+		secretRefName := GetEnv(ENV_PREDICTIVE_UNIT_DEFAULT_ENV_SECRET_REF_NAME, "")
+		if secretRefName == "" {
```

Comment:
Should the logic not be the other way around? If the `pu.EnvSecretRefName == ""` then get from EnvVar otherwise use the one the user specified in the graph explicitly.

Reply:
Yeah good catch - will add a unit test to ensure it's enforced / chacked